evalai/utils/challenges.py

```diff
@@ -174,3 +175,51 @@ def display_participated_or_hosted_challenges(is_host=False, is_participant=Fals
                 pretty_print_challenge_data(challenge)
         else:
             echo("Sorry, no challenges found!")
+
+
+def pretty_print_leaderboard_data(attributes, results):
+    """
+    Pretty print the leaderboard for a particular CPS.
+    """
+    leaderboard_table = BeautifulTable()
+    attributes = ["Rank", "Team"] + attributes + ["Last Submitted"]
+    leaderboard_table.column_headers = attributes
+
+    for count, result in enumerate(results):
```

Comment:
You can pass in the argument `start=1` to enumerate so that `count` starts from 1. That way you can rename count to rank.

Reply:
@varunagrawal  Done! :D